core/src/mindustry/entities/comp/UnitComp.java

```diff
@@ -485,7 +485,7 @@ public void destroy(){
         }
 
         //if this unit crash landed (was flying), damage stuff in a radius
-        if(type.flying && !spawnedByCore){
+        if(type.flying && !spawnedByCore && team == state.rules.defaultTeam){
```

Comment:
maybe && team != TowerDefense.attackingTeam
instead?

Reply:
Also is attackingTeam necessary? You can just use state.rules.waveTeam since the attacking team will be the one with waves

Reply:
That also makes it so it still works in case the wave team is different in the map, since settings can change it now